src/util/Template.ts

```diff
@@ -0,0 +1,8 @@
+export function renderTemplate(template: string, args: { [key: string]: string }): string {
```

Comment:
Does this need a copyright clause like in some other files? @Half-Shot 

Most seem to have it, but PackageInfo.ts does not, for instance.

Reply:
Yes, ones that don't are bad and wrong.

Reply:
Fixed in 6b6146bf. Should we add a linter rule/CI script that'll check for that? (Probably in a separate task).